scripts/testpop/statter.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+)
+
+// WriteCloserToSender is an adapter from io.WriteCloser to statsd.Sender
+type WriteCloserToSender struct {
+	bufWriter *bufio.Writer
+	closer    io.Closer
+}
+
+// Send implements the statsd.Sender interface
+func (w *WriteCloserToSender) Send(data []byte) (int, error) {
+	return w.bufWriter.Write(data)
```

Comment:
Don't we need to flush the writer every time we want to send, else I don't really get why we'd use a buffered writer